src/lib/__tests__/server-utilities.test.js

```diff
@@ -0,0 +1,18 @@
+import { md5Hash } from 'lib/server-utilities';
+jest.unmock('lib/server-utilities');
+
+describe('md5Hash', () => {
+  const adminClientHash = md5Hash('./static/build/editor-client.js');
+  it('returns a string', () => {
+    expect(typeof adminClientHash).toBe('string');
+  });
+
+  it('hashes consistently', () => {
+    const secondAdminClientHash = md5Hash('./static/build/editor-client.js');
+    expect(adminClientHash).toBe(secondAdminClientHash);
+  });
```

Comment:
> Also, another good test would be testing that the string ./static/build/editor-client.js isn't being hashed, but rather the file that the path references is what is being hashed.

That's a good idea :). Because it's exactly good form to always write a test for a problem you encountered to make sure you don't encounter it again, or future changes run into it again :). 

And generally sure thing, good ideas for simple tests that at least show that the hash isn't just a constant value.

I think I'll also maybe just make a test file or use the src files or something, as I also realized that probably the tests would fail at the moment if you haven't run a build command yet as it's trying to hash build files that don't come with the git repo.